report/src/statChart.ts

```diff
@@ -0,0 +1,212 @@
+            case "normal":
+                return [this.stdDev(-4), this.stdDev(4)];
+            case "min_max":
+            default:
+                return [this.quartile(0), this.quartile(4)];
+        }
+    }
+
+    enter(domNode, element) {
+        super.enter(domNode, element);
+
+        this._bellCurveDiv = element.append("div");
+        this._bellCurve.target(this._bellCurveDiv.node());
+
+        this._candleDiv = element.append("div");
+        this._candle.target(this._candleDiv.node());
+
+        const checkDiv = element.append("div")
+            .style("position", "absolute")
+            .style("top", "0px")
+            .style("right", "0px")
+            ;
+        this._stdViewCheckbox = checkDiv.append("select")
+            .on("change", () => this.render())
+            ;
+        this._stdViewCheckbox.append("option").attr("value", "min_max").text("Min / Max");
+        this._stdViewCheckbox.append("option").attr("value", "25_75").text("25% / 75%");
+        this._stdViewCheckbox.append("option").attr("value", "normal").text("Normal");
+    }
+
+    private bellTicks(mode: Mode): AxisTicks {
+        let ticks: Ticks;
+        switch (mode) {
+            case "25_75":
+                ticks = [
+                    { label: this.formatQ(1), value: this.quartile(1) },
+                    { label: this.formatQ(2), value: this.quartile(2) },
+                    { label: this.formatQ(3), value: this.quartile(3) }
+                ];
+                break;
+            case "normal":
+                ticks = [
+                    { label: this.formatStdDev(-4), value: this.stdDev(-4) },
+                    { label: "-3σ", value: this.stdDev(-3) },
+                    { label: "-2σ", value: this.stdDev(-2) },
+                    { label: "-1σ", value: this.stdDev(-1) },
+                    { label: this.formatStdDev(0), value: this.stdDev(0) },
+                    { label: "+1σ", value: this.stdDev(1) },
+                    { label: "+2σ", value: this.stdDev(2) },
+                    { label: "+3σ", value: this.stdDev(3) },
+                    { label: this.formatStdDev(4), value: this.stdDev(4) },
+                ];
+                break;
+            case "min_max":
+            default:
+                ticks = [
+                    { label: this.formatQ(0), value: this.quartile(0) },
+                    { label: this.formatQ(1), value: this.quartile(1) },
+                    { label: this.formatQ(2), value: this.quartile(2) },
+                    { label: this.formatQ(3), value: this.quartile(3) },
+                    { label: this.formatQ(4), value: this.quartile(4) }
+                ];
+        }
+
+        const [domainLow, domainHigh] = this.domain(this._stdViewCheckbox.node().value);
+        return ticks
+            .filter(sd => sd.value >= domainLow && sd.value <= domainHigh)
+            .map(sd => { return { label: sd.label, value: sd.value.toString() }; })
```

Comment:
Should use more than one line since `return` is used.

Reply:
Changed to use correct shorthand